packages/apptive_grid_core/lib/network/apptive_grid_client.dart

```diff
@@ -270,26 +272,12 @@ class ApptiveGridClient {
   /// Load Entities of a Grid that are accessed by [uri]
   /// [viewId] specifies allows to request the entities of a specific GridView which allows for getting Filters/Sorting that was applied from the web client
   ///
-  /// if [layout] is not set the items will be a Map<String, dynamic> with the keys being the fieldIds and the value being the values as json objects
-  /// `indexed` layout will return the items in the following format:
-  /// ```
-  /// {
-  ///   "fields": [
-  ///     `jsonValue of First Column`,
-  ///     `jsonValue of Second Column`,
-  ///     `jsonValue of Third Column`,
-  ///     `jsonValue of Fourth Column`,
-  ///     ...
-  ///   ]
-  /// }
-  /// ```
-  ///
```

Comment:
Shouldn't there still be some kind of docu?

Reply:
This is concerning the layout parameter for loadEntities, it  kind of cut off 😅

Reply:
Re-added a line and added something to the single call as well